controller/users_blackbox_test.go

```diff
@@ -984,3 +1009,125 @@ func createDummyUserProfileResponse(updatedBio, updatedImageURL, updatedURL *str
 	return profile
 
 }
+
+func (s *TestUsersSuite) TestCreateUserAsServiceAccountOK() {
+
+	// given
+	user := testsupport.TestUser
+	identity := testsupport.TestIdentity
+	identity.User = user
+	identity.ProviderType = "KC"
+
+	user.ContextInformation = map[string]interface{}{
+		"last_visited": "yesterday",
+		"space":        "3d6dab8d-f204-42e8-ab29-cdb1c93130ad",
+		"rate":         100.00,
+		"count":        3,
+	}
+	user.Company = "randomCompany"
+	secureService, secureController := s.SecuredServiceAccountController(testsupport.TestOnlineRegistrationAppIdentity)
+
+	// when
+	createUserPayload := createCreateUsersAsServiceAccountPayload(&user.Email, &user.FullName, &user.Bio, &user.ImageURL, &user.URL, &user.Company, &identity.Username, &identity.RegistrationCompleted, user.ContextInformation, user.Cluster)
+	_, appUser := test.CreateUsersOK(s.T(), secureService.Context, secureService, secureController, createUserPayload)
+	assertCreatedUser(s.T(), appUser.Data, user, identity)
```

Comment:
Check if user is created correctly for a payload:
- with required fields only
- with all fields

Also check if user creation fails if some required info is missing/empty - username, email, etc.